### ISSUE TITLE
Can't detect my four fingers on gnome wayland

### ISSUE THREAD

**Issue by leewdch** [bulletmark/libinput-gestures#166]
```
libinput-gestures: session unknown+wayland on Linux-4.17.9-1-ARCH-x86_64-with-arch, python 3.6.6, libinput 1.11.2
Gestures configured in ~/.config/libinput-gestures.conf:
swipe left       4 _internal ws_up
swipe right      4 _internal ws_down
libinput-gestures: device /dev/input/event18: SynPS/2 Synaptics TouchPad
```

I'm trying configuring 4 finger gestures to switch workspaces in gnome wayland but when I try testing gestures as suggested in the documentation libinput says I'm using 3 fingers when I'm using 4 instead

![image](https://user-images.githubusercontent.com/11262717/43278030-1d35303e-910a-11e8-91ea-abcc2c7d3b8d.png)

**Comment by bulletmark** [comment]
There are a number of points to make here:

1. You have stated "on gnome wayland" prominently in your issue title implying that this works "on gnome Xorg".  Does it? I doubt it makes any difference.

2.  Did you follow the TROUBLESHOOTING section? It says to run `libinput-gestures -r` as raw mode to see what libinput is directly reporting. Probably your touchpad does not report 4 finger gestures at all, many touchpads do not, including my own. Convince yourself of this by completely shutting down `libinput-gestures` and by running `libinput debug-events`. I.e. it has nothing to do with this utility.

3. You configuration shows you are merely trying to get swipe up and down working with 4 fingers in GNOME Wayland. Did you read WAYLAND AND OTHER NOTES section? As it says, Wayland already supports that exact gesture natively so you don't need `libinput-gestures` at all (well if your touchpad reported 4 finger gestures).

4. You seem to have an odd GNOME setup because it is being reported as session type "unknown" in the above output.

5. You pasted the first screen text output directly and then pasted the second screen output as an image file. Why paste an image for simple text output, and then do it two different ways?

**Comment by leewdch** [comment]
1. I specified what I was using because I thought it would matter
2. You are right, raw mode only detects 3 fingers but I used the same configuration on xorg+bspwm and it correctly detected 4 fingers since I used 3 fingers swipes to go prev and next in the browser and 4 fingers swipes to switch workspace
3. I didn't find a way to set the config for the gestures so I stuck with the only method I knew, libinput-gestures
4. I don't really know how to reply to this. I just installed the gnome package and launched it at start with wayland
```
if [[ -z $DISPLAY ]] && [[ $(tty) = /dev/tty1 ]] && [[ -z $XDG_SESSION_TYPE ]]; then
  XDG_SESSION_TYPE=wayland exec dbus-run-session gnome-session
fi
```
5. The output from the text I pasted and the output from the screenshot are from two different flag options. I merely used the command suggested in the new issue's placeholder and the command I generally use to debug loaded configurations
